cli/src/config.rs

```diff
@@ -71,6 +73,7 @@ impl Default for Configuration {
             public_key,
             private_key,
             disable_panic_terminal_colors: bool::default(),
+            shutdown_on_panic: false,
```

Comment:
So we're effectively disabling the panic hook. This is useful as a stop-gap solution, but I would prefer resolving the problem.